packages/runc/Cargo.toml

```diff
@@ -9,8 +9,9 @@ build = "build.rs"
 path = "pkg.rs"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/opencontainers/runc/archive/b9ee9c6314599f1b4a7f497e1f1f856fe433d3b7/runc-b9ee9c6314599f1b4a7f497e1f1f856fe433d3b7.tar.gz"
-sha512 = "ebc67bbc825aa6d7a51e9504d3b4f41794861c06a18288395c035c1b6929e9fb1bae0dcf1b14d9120b651f4a63fdcd6ce8e575a2e84bf819d0d0d1d1cb4e2bae"
+url = "https://github.com/opencontainers/runc/releases/download/v1.0.1/runc.tar.xz"
+path = "runc-v1.0.1.tar.xz"
```

Comment:
Can we use the same `#/desired-name.ext` syntax as in the `Source` line of the spec file, rather than needing a `path` override here?

Reply:
We'd have to update [extract_file_name](https://github.com/bottlerocket-os/bottlerocket/blob/b9c8747e7abf4f5dd02e217571ed8d26f3f3db72/tools/buildsys/src/cache.rs#L101) to prefer a fragment if present.

I'm not sure how I feel about that. It would let us remove the "path" setting and align `Cargo.toml` with the spec file URI, which seems good. But it's also more cryptic.